core/frontend/src/components/wizard/Wizard.vue

```diff
@@ -210,8 +210,8 @@ export default Vue.extend({
     },
   },
   async mounted() {
-    const wizard = await bag.getData('wizard')
-    if (wizard?.version !== WIZARD_VERSION) {
+    const wizard = await bag.getData('wizard2')
+    if (wizard !== undefined && wizard?.version !== WIZARD_VERSION) {
```

Comment:
what is the logic here? is undefined for "unable to reach backend"? I think it deserves at least a comment